README.md

```diff
@@ -1 +1,183 @@
-# budget-app
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+
+  <img src="https://github.com/rbhatt1999/rbhatt1999/blob/bd8bb27a9814ab74d6f920a1027494a65610a24d/logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Budget App</b></h3>
+</div>
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget App ](#-budget-app-)
+  - [🛠 Built With ](#-built-with-)
+  - [🛠 Tech Stack ](#-tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo](#live-demo)
+  - [🚀 Video Presentation](#presentation)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors](#-authors)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget-app <a name="about-project"></a>
+
+Budget app is an app for managing transactions with categorized lists, sign-up and login pages, and navigation buttons for moving between pages. Users can add new categories or transactions.
+
+## 🛠 Built With <a name="built-with"></a>
+This app is build with Ruby on Rails and Postgresql.
+### Tech Stack <a name="tech-stack"></a>
+> <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+> <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+> <li><a href="https://www.postgresql.org/">Postgresql</a></li>
+
+### Key Features <a name="key-features"></a>
+
+> - Categorized lists.
+> - Navigation buttons for moving between pages.
+> - Ability to add new categories with mandatory name and icon fields.
+> - Ability to add new transactions with mandatory name, amount, and category fields.
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-6lkf.onrender.com/)- Powered by Render
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Video Presentation <a name="presentation"></a>
+
+- [Video Presentation Link](https://www.loom.com/share/a57474cdf4d04160b19374afad20f577)- Powered by Loom
```

Comment:
Great presentation video 👏 

One thing is left though, you are required to show a piece of code you're proud of, explain it and tell us why you are proud of it.

Reply:
https://www.loom.com/share/865d4fa527cd410db11f4f2931f4cc98